README.adoc

```diff
@@ -252,6 +252,14 @@ The PCT revision to use in case that pctUrl points to a github destination, can
  A String indicating the file path (relative to where this step is executed) to use as metadata file for the build, more details about the metadata file are provided belows. *Defaults to `essentials.yml` at the location where this step is invoked*
 `jenkins`::
  URI to the jenkins.war, Jenkins version or one of "latest", "latest-rc", "lts" and "lts-rc". Defaults to "latest". For local war files use the file:// protocol in the URI. *Can be overriden from the metadata file*
+`pctExtraOptions`:: 
+ List of extra PCT options to be passed to the PCT executable. Defaults to empty list.
+`javaOptions`:: 
+ List of extra Java options to be passed to the PCT executable. Defaults to empty list.
+`dockerOptions`::
+ List of extra options to be passed to PCT containers ( e.g. `maven-repo:/root/.m2`)
+`useJDK11`::
```

Comment:
Noted. I need to change https://github.com/jenkinsci/plugin-compat-tester/pull/86 as well to reflect that.

Reply:
PCT is fine IMHO - it is a low-level tool. I doubt we will keep packaging all JDK versions into a single image going forward

Reply:
I said that because at the moment, the JDK selection in PCT is done by the existance of the `JDK11` env var. When this env var exists, it sets the JDK path and some argument (modules, etc). I did that to have a "easy" java 11 configuration on PCT